scripts/patch_auditwheel_recursive_dependency_bug.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# This is a patch to handle auditwheel's bug with recursive depenencides:
+# https://github.com/pypa/auditwheel/issues/48
+
+sed -i 's/if is_py_ext:/if True:/g' /auditwheel/wheel_abi.py
```

Comment:
Use `patch here` maybe? Instead of sed. Create wheel_abi.patch with `diff` and use it